### PR TITLE
Make corrections in Raytune MR-gRPC demo example notebook

### PR DESCRIPTION
**Pull request by abhijeet-dhumal**
## Description

Make corrections in Raytune MR gRPC demo example notebook

## How Has This Been Tested?

Manually tested - [demo recording](https://drive.google.com/file/d/1nAaHpo7aj4R6lFN_j-qMeTEh8OTCuVwU/view?usp=sharing)

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by astefanutti — COMMENTED** [review]
Could you add the MinIO deployment that you've used, I think it would be useful as an example on how to deploy a local S3.

**Comment by abhijeet-dhumal** [comment]
> Could you add the MinIO deployment that you've used, I think it would be useful as an example on how to deploy a local S3.

Hey @astefanutti, 
Do you mean to add minio-deployment file in hpo-raytune directory as a sample and to mention steps performed in readme for deploying it ?

**Comment by astefanutti** [comment]
> > Could you add the MinIO deployment that you've used, I think it would be useful as an example on how to deploy a local S3.
> 
> Hey @astefanutti, Do you mean to add minio-deployment.yaml file in hpo-raytune directory as a sample and to mention steps performed in readme for deploying it ?

Yes, exactly, and mentioning somewhere in the doc that it's an option for users if they don't have S3 storage readily available.

File: examples/hpo-raytune/README.md [hunk 1]
```diff
@@ -35,6 +35,19 @@ This example primarily focuses on optimizing machine learning models using RayTu
 * Deploy a model serving inference using an endpoint with the following details:
 ![](./docs/model-serving-inference.png)
 
+
+### Setting Up Local S3 Compatible Storage - Minio
+
+To create local S3 compatible storage type using Minio, Follow these steps:
```
Comment (astefanutti) [inline examples/hpo-raytune/README.md:41]: I'd suggest to start with something like "If you don't have an S3 storage readily available, you can create a local ...".

File: examples/hpo-raytune/README.md [hunk 2]
```diff
@@ -35,6 +35,19 @@ This example primarily focuses on optimizing machine learning models using RayTu
...
+
+1. Login to Openshift cluster
+
+2. Run the following command to apply minio deployment yaml file
+
+    ```bash
+    kubectl apply -f ./resources/setup-minio.yaml
+    ```
+
```
Comment (astefanutti) [inline examples/hpo-raytune/README.md:50]: Is there any post deployment actions needed, like creating a storage?

Comment (abhijeet-dhumal) [reply examples/hpo-raytune/README.md:50]: Yes, there  is a manual step which can be added in this case..
Thank you @astefanutti for reviewing !

**Comment by sutaakar** [comment]
@abhijeet-dhumal is the PR ready for merge?

**Review by astefanutti — APPROVED** [review]

**Pull request merged**